server/controllers/api.py

```diff
@@ -151,8 +139,7 @@ def get(self, user):
         return {
             'version': API_VERSION,
             'url': '/api/{}/'.format(API_VERSION),
-            'documentation': 'http://github.com/Cal-CS-61A-Staff/ok/wiki',
-            'access_token': session['google_token']  # FOR TESTING ONLY
```

Comment:
No need. It was marginally useful while working on the token auth but I don't think it's needed here. 
In an ideal world our users should never had to deal with their access_token